tests/harness.c

```diff
@@ -322,6 +332,23 @@ bool push_bootloader_update(int revision)
                 fprintf(stderr, "Failed to update bootloader: %s\n", strerror(errno));
                 return false;
         }
+#if defined(HAVE_SHIM_SYSTEMD_BOOT)
+        {
```

Comment:
bracket needed here? seems cosmetic afaics

Reply:
it's just to have scope for an isolated task and its variables declared where they belong (both in scope and the beginning of it). is it really an issue?